hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/statistics/IOStatisticsContext.java

```diff
@@ -80,4 +80,14 @@ static void setThreadIOStatisticsContext(
     IOStatisticsContextIntegration.setThreadIOStatisticsContext(
         statisticsContext);
   }
+
+  /**
+   * Static probe to check if the thread-level IO statistics enabled.
+   *
+   * @return if the thread-level IO statistics enabled.
+   */
+  static boolean getIOStatisticsThreadLevelEnabled() {
```

Comment:
not sure this is the right predicate. i think a simple `enabled()` would be informative enough, as what is being enabled is obvious